src/main/java/ch/njol/skript/expressions/ExprCharges.java

```diff
@@ -0,0 +1,90 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.type.RespawnAnchor;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import static java.lang.Math.max;
+import static java.lang.Math.min;
+
+public class ExprCharges extends SimplePropertyExpression<Block, Number> {
+
+	static {
+		if(Skript.isRunningMinecraft(1, 16)) {
+			register(ExprCharges.class, Number.class, "charge[s]", "blocks");
+		}
+	}
+
+	@Nullable
+	@Override
+	public Number convert(Block block) {
+		if(!block.getBlockData().getMaterial().equals(Material.RESPAWN_ANCHOR)) {
+			Skript.error("You can only use the 'charges' expression with a respawn anchor!");
+			return null;
+		}
+		RespawnAnchor anchor = (RespawnAnchor) block.getBlockData();
+		return anchor.getCharges();
+	}
+
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
+		if (mode == Changer.ChangeMode.SET)
+			return CollectionUtils.array(Number.class);
+		return null;
+	}
+
+	@Override
+	public void change(Event e, @Nullable Object[] delta, Changer.ChangeMode mode) {
```

Comment:
These are numbers being changed in the change method, All ChangeModes can be supported aside from the optional ChangeMode.REMOVE_ALL. I made an example write up in another pull request here https://github.com/SkriptLang/Skript/pull/4854#pullrequestreview-1024714523

Rather than importing Changer.ChangeMode mode this can just be ChangeMode mode. There is no need to import the whole Changer class, we just need the ChangeMode enum from the Changer class.